test/integration/nsmd_test_utils/test_utils.go

```diff
@@ -523,7 +523,7 @@ func IsMemifNsePinged(k8s *kube_testing.K8s, from *v1.Pod) (result bool) {
 	}
 	logrus.Infof("nse ip: %v", nseIp)
 	logrus.Infof(" %v trying vppctl ping to %v", from.Name, nseIp)
-	for attempts := 30; attempts > 0; <-time.Tick(300 * time.Millisecond) {
+	for attempts := 60; attempts > 0; <-time.Tick(300 * time.Millisecond) {
```

Comment:
Ugh, we need more attempts in order to succeed. Should not hurt in general, but is there a bigger problem hiden here? Or is it just the instances are slow?

Reply:
Effectively this is yet another increased timeout. The reason is the same as for the `defaultTimeout`. So nothing new here. 

On the other hand setting up memif interface takes surprisingly long time on all platforms. I think we should take a look at the memif creation performance. Denis Tingajkin is working on memif performance tests, as I know.